packages/cortex-tutor/tools.py

```
@@ -0,0 +1,493 @@
+                    "success": True,
+                    "cache_hit": True,
+                    "lesson": cached,
+                    "cost_saved_gbp": 0.02,
+                }
+
+            return {
+                "success": True,
+                "cache_hit": False,
+                "lesson": None,
+                "reason": "No valid cache found",
+            }
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)